src/protocol.h

```diff
@@ -263,7 +263,7 @@ enum ServiceFlags : uint64_t {
     NODE_BLOOM = (1 << 2),
     // NODE_WITNESS indicates that a node can be asked for blocks and transactions including
     // witness data.
-    NODE_WITNESS = (1 << 3),
```

Comment:
It might make more sense to define a new version bit, like in the BIP149 reference implementation:

https://github.com/bitcoin/bitcoin/compare/master...shaolinfry:uasegwit-flagday#diff-1df47808b82268c616535e7b20216dfbR272